### ISSUE TITLE
Can you add Port as a config option for the MSSQL db

### ISSUE THREAD

**Issue by dennisnotojr** [redconnect-io/node-red-contrib-mssql#25]
Several instances where people don't use the well know port and would like to set it in the configuration.  Would be a big help. Den

**Comment by jose-miranda-tierconnect-com** [comment]
Yes please that will be a great help. to have a configurable port.

**Comment by sap-auto** [comment]
you can add port by adding a text input form in mssql.html and then adding a few lines in mssql.js to map that to port.

To start with navigate to the directory after installing node-red-contrib-mssql
 cd $HOME/.node-red/
 cd node_modules
 cd  node-red-contrib-mssql

Create a backup of the original files



 cp mssql.html mssql.html.bak
 cp mssql.js mssql.js.bak

open mssql.js in text editor 
nano mssql.js

and add the following line after "server: config.server,"
port: config.port,

open mssql.html in text editor 
nano mssql.html

and add the following line in the form where you want the port input
```
<div class="form-row">
        <label for="node-config-input-port"><i class="icon-bookmark"></i> Server</label>
        <input type="text" id="node-config-input-port" placeholder="Server name Port">
    </div>
```

Screenshot attached
![mssql_html_change](https://user-images.githubusercontent.com/3961329/38018041-6f36e2c4-3291-11e8-8638-a0fa5e7decd5.jpg)
![mssql_js_change](https://user-images.githubusercontent.com/3961329/38018042-6f7a3c54-3291-11e8-8bd7-f84eec25b0f3.jpg)
